### DISCUSSION TITLE
Audio from the Lessons

### DISCUSSION THREAD

**Discussion by rschmidtecm** [SethClydesdale/genki-study-resources#77] [Questions]
I there any plan to include the audio samples from the main lessons in addition to the workbook audio?  Excellent job, by the way.

**Comment by SethClydesdale** [comment]
Hello @rschmidtecm,

Thank you!

Probably not, since it would add exponentially to the size of the repository which I'd like to keep around <100mb so downloading the repo remains quick for offline users. All audio together is around 700mb and probably 500mb using current compression methods, multiply that by 2 (Genki 2nd & 3rd edition) and it'll roughly come out to about 1gb which is the size limit for GitHub repos. 

The other thing I'd like to avoid is legal issues for fully distributing the audio files, so I'd only like to use those which are absolutely necessary, such as for the listening comprehension exercises. Other audio would have to be used on the user end via the CD/MP3 or OTO Navi application.

If you have anymore questions, feel free to let me know.

Have a good day.

**Comment by SethClydesdale** [comment]
Hello,

As it has been a month without activity on this issue, I'll mark it closed for now. Feel free to let me know if you would like to reopen this issue.
